activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -14,7 +14,7 @@ module FinderMethods
     #   Person.find(1, 2, 6)    # returns an array for objects with IDs in (1, 2, 6)
     #   Person.find([7, 17])    # returns an array for objects with IDs in (7, 17)
     #   Person.find([1])        # returns an array for the object with ID = 1
-    #   Person.where("administrator = 1").order("created_on DESC").find(1)
+    #   Person.where("administrator = 1").order("created_on DESC").find(1) # returns the object for ID = 1
```

Comment:
Actually we can't tell this. It may raise record not found because the object with id 1 is not administrator. As it is hard to tell what is the expected result I'd just keep it as it is.